core/cmd/local_client_test.go

```diff
@@ -207,8 +205,6 @@ func TestClient_ImportKey(t *testing.T) {
 }
 
 func TestClient_LogToDiskOptionDisablesAsExpected(t *testing.T) {
-	t.Parallel()
```

Comment:
This test, running in parallel, changed the logger to `json` which ruined the assertions in `TestClient_RunNodeShowsEnv`. Simply removing this test's parallelism should fix the spurious failures.